app/application.js

```diff
@@ -88,11 +88,11 @@ class RunConditionTableApplication {
             this.logger.info('Stopping RCT app...');
             try {
                 await this.httpServer.close();
+                this.rl.close();
```

Comment:
Same here, plus `rl` require to look for definition to understand what this function does.